content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -242,9 +243,7 @@ PVCs indirectly if they can create Pods, even if they do not have
 permission to create PVCs directly. Cluster administrators must be
 aware of this. If this does not fit their security model, they have
 two choices:
-- Explicitly disable the feature through the feature gate, to avoid
-  being surprised when some future Kubernetes version enables it
-  by default.
+- Explicitly disable the feature through the feature gate.
 - Use a [Pod Security
   Policy](/docs/concepts/policy/pod-security-policy/) where the
   `volumes` list does not contain the `ephemeral` volume type.
```

Comment:
For the quota discussion below, I would link to our pages for storageclass quota.

Reply:
Added a link to https://kubernetes.io/docs/concepts/policy/resource-quotas/#storage-resource-quota